addons/dogtags/XEH_postInit.sqf

```diff
@@ -46,5 +46,35 @@ if (["ACE_Medical"] call EFUNC(common,isModLoaded)) then {
     };
 };
 
+if (["ACE_Arsenal"] call EFUNC(common,isModLoaded)) then {
```

Comment:
There is no need for this, the event simply won't be ran if ACE Arsenal is not loaded.